IBX-2889: Cross content type recommendations (#141)
ezsystems/user-documentation: docs/personalization/cross_content_type_reco.md

# Cross content type recommendations

> Cross content type option is used to get multiple recommendation items for each input types. 

Reviewer: ```suggestion
Cross content type option is used to get multiple recommendation items for each output types. 
```

# Cross content type recommendations

Cross content type option is used to combine best recommendation items from different content types.
It applies to scenarios which have more than one output type configured.

## Configure scenario

To get multiple output types in the recommendation request, perform the following:

1. Go to **Scenarios**.
1. Click the **Edit** icon next to the scenario you want to set cross content type recommendations.
1. From the **Output type** multiselect field, select the types you want to get recommendations for in the request.
    It contains a list of all content types exported for the specified customer ID.
1. Next, go to Preview Scenario.
1. From the **Output type** drop-down, select **All**. 

> The **All** option automatically appears after content types are selected in the scenario settings.

Reviewer 2: It seems that if one has selected several output types in the edit phase, the "All" option is selected by default when you go to. preview, so there is no need to select it. You could say something like...

``` suggestion
1. From the **Output type** drop-down, select the outputs for which you want to generate the preview. 
```

And then, the sentence about the "All" option makes more sense:

"After you select content types in the scenario settings, the **All** option automatically appears here and, by default, it is selected."

However, on a second thought, it is describing the inner workings of the system, so why not simply:

"By default, the **All** option is selected."

!!! note

    The **All** option can be translated according to your needs.

![Cross content type](img/perso_cross_content_type.png)

## Use cross content type in Page Builder blocks

> When scenarios are configured to display cross content type recommendations, you can use them in the two Page Builder blocks: Dynamic targeting and Personalized.

Reviewer 2: ```suggestion
When scenarios are configured to display cross content type recommendations, you can use them in the following Page Builder blocks: Dynamic targeting and Personalized.
```

> To get all output types in the [Personalized block](../site_organization/working_with_page.md#personalized-block), in the Page Builder mode perform the following:

Reviewer 2: ```suggestion
To get all output types in the [Personalized block](../site_organization/working_with_page.md#personalized-block), in Page Builder, perform the following actions:
```

> 1. In the block settings, set the scenario with configured crosscontent types output.
> 1. Next, from the drop-down **Select a Content Type to be displayed**, select **All**.
> 1. Increase the display limit to make sure all recommendations are shown.
> 1. Click **Submit**.

Reviewer 2: ```suggestion
1. Click the **Edit** icon to display the settings of the **Personalized** block.
1. Select a scenario with cross content types output configured.
1. From the **Select a Content Type to be displayed** drop-down, select **All**.
1. Increase the display limit to ensure that all recommendations are shown.
1. Click **Submit**.
```

To get all output types in the [Dynamic targeting block](../site_organization/working_with_page.md#dynamic-targeting-block):

> 1. In the block settings, set the scenario with configured crosscontent types output.
> 1. From the **Output type** drop-down, select **All**.
> 
> 1. Next, set the rules according to your needs.
> 1. Click **Submit**.

Reviewer 2: ```suggestion
1. Click the **Edit** icon to display the settings of the **Dynamic targeting** block.
1. Select a scenario with cross content types output configured.
1. From the **Output type** drop-down, select **All**.
1. Set the matching rules according to your needs.
1. Click **Submit**.
```
